lib/schema/viewcompiler.js

```diff
@@ -54,14 +54,16 @@ class ViewCompiler {
       (materialized ? 'materialized ' : '') +
       'view ' +
       (replace ? 'or replace ' : '');
-    const formatColumns = [];
-    for (const c of columns) {
-      formatColumns.push(
+    const formatColumnsArr = [];
```

Comment:
You can use the same syntax from mssql:

```js
const columnList = columns
      ? ' (' +
        columnize_(
          columns,
          this.viewBuilder,
          this.client,
          this.bindingsHolder
        ) +
        ')'
      : '';
```

last comment, all is fine for me :D. Thanks again.